script/test_todo.py

```diff
@@ -46,6 +46,31 @@ def verify_all_completed_task_are_deleted(driver):
     assert elem.get_attribute("class") != "completed"
 
 
+def click_tab_active(driver):
```

Comment:
Create one function `click_tab` which will take as a parameter tab which should be clicked.